polypolish/0.6.0/Dockerfile

```diff
@@ -0,0 +1,51 @@
+ARG POLYPOLISH_VER="0.6.0"
+
+FROM rust:1.75 as builder
+
+ARG POLYPOLISH_VER
+
+RUN wget -q https://github.com/rrwick/Polypolish/archive/refs/tags/v${POLYPOLISH_VER}.tar.gz && \
+    tar -vxf v${POLYPOLISH_VER}.tar.gz && \
+    cd /Polypolish-${POLYPOLISH_VER} && \
+    cargo build --release 
+
+FROM ubuntu:jammy as app
+
+ARG POLYPOLISH_VER
+
+COPY --from=builder /Polypolish-${POLYPOLISH_VER}/target/release/polypolish /usr/local/bin/polypolish
```

Comment:
It is better to have the COPY line after installing dependencies to maintain the same style as other Dockerfiles.

Reply:
Just asking out of curiosity; is the prebuilt binary not working on Jammy?